autode/species/species.py

```diff
@@ -228,6 +229,7 @@ def find_lowest_energy_conformer(self, lmethod=None, hmethod=None):
             hmethod (autode.wrappers.ElectronicStructureMethod):
         """
         logger.info('Finding lowest energy conformer')
+        methods.add('Low energy conformers located with the')
```

Comment:
should this be the start if the methods_string, instead of adding to methods now?

Reply:
self._generate_conformers() adds to the methods, but this line should definitely after the check on 2 atoms